src/functions-and-arrays.js

```diff
@@ -1,19 +1,85 @@
 // Iteration #1: Find the maximum
+function  maxOfTwoNumbers(x1,x2){
+  if (x1>x2) {
+    return x1;
+  } else if(x2>x1) { return x2;
+  } else {
+    return (x1,x2)
+  } 
+  }
 
+console.log(maxOfTwoNumbers(3,8))
 
 // Iteration #2: Find longest word
-const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
+const words=['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
+
+function findLongestWord(arr) {
+ if (arr.length===0){
+  return null;
+  } else {
+
+    let longestWord = "";
+    
+    for (let i = 0; i < arr.length; i++) {
+      if (arr[i].length > longestWord.length) {
+        longestWord = arr[i];
+      }
+      return longestWord;
+    
+    }
```

Comment:
Lembre-se! o `return` ao ser usado encerra a função imediatamente!
Então colocando o `return logestWord` dentro do ´for` resulta nele retornando sempre só a primeira palavra, independente de tamanho!
Basta colocar o `return logestWord` **depois** do `for`!